.github/workflows/stale.yml

```diff
@@ -0,0 +1,18 @@
+name: "Mark stale PRs"
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-pr-message: 'There was no activity on this pull request for a significant amount of time. It will be closed soon. Please reach out to an editor, if you are still pursuing it.'
```

Comment:
We should probably let them know *how* to reach out to an editor.  For me, just commenting on the pull request is enough to get my attention.  I can't speak for the process of the other editors but if I was the only editor I would want the messaging to go something like:
> There has been no activity on this pull request for two months.  It will be closed in a week if no further activity occurs.  If you would like to move this EIP forward, please respond to any outstanding feedback or add a comment indicating that you have addressed all *required* feedback and are ready for a review.

Reply:
```suggestion
        stale-pr-message: 'There has been no activity on this pull request for two months. It will be closed in a week if no further activity occurs. If you would like to move this EIP forward, please respond to any outstanding feedback or add a comment indicating that you have addressed all required feedback and are ready for a review.'
```

Reply:
Actually the new wording suggests a pull request can only be a new EIP, but there are plenty of modifications pending author's approval. Not sure how much lengths we have to go to get this right though 😕